docker/client.py

```diff
@@ -239,7 +241,7 @@ def _multiplexed_response_stream_helper(self, response):
         # Disable timeout on the underlying socket to prevent
         # Read timed out(s) for long running processes
         socket = self._get_raw_response_socket(response)
-        if six.PY3:
+        if six.PY3 and hasattr(socket, "_sock"):
```

Comment:
Sorry, to clarify: I'm fine with the way it works now, I'm just wondering about the difference between this line and [that one](https://github.com/docker/docker-py/pull/594/files#diff-5262437d121a30fc85a81c4b1da57e4bR186). I like testing for the existence of `_sock` better than testing the format of `base_url`.